External/FEXCore/Source/Interface/Core/LLVMJIT/LLVMCore.cpp

```diff
@@ -184,6 +197,10 @@ class LLVMJITCore final : public CPUBackend {
     Result = JITState.IRBuilder->CreateShuffleVector(Result, ZeroVector, VectorMaskConstant);
 
 #elif _M_ARM_64
```

Comment:
I'm skeptical of these defines - why are they needed?

Reply:
These are needed because we are actually using architecture specific intrinsics that behave slightly differently depending on x86 or aarch64.
Sadly LLVM doesn't have any idiom detection for automatically choosing the `packusdw` or `sqxtun{2}` instructions.